src/equipment_inventory/admin.py

```diff
@@ -39,7 +39,7 @@ class EquipmentAdmin(admin.ModelAdmin):
     form = EquipmentForm
 
     def response_change(self, request, obj):
-        return redirect(reverse('equipment-list', kwargs={'pk': obj.pk}))
+        return redirect(reverse('equipment-list'))
```

Comment:
In other views, we are redirecting back to the detail view of the object that was edited. We need to decide if we want to redirect to a list view or detail view after saving an edit.

Reply:
after editing something, the form should redirect you to the details page of that something, and when you're deleting it should redirect you to the list view